src/repositories/neptune.js

```diff
@@ -0,0 +1,57 @@
+const neptune = require("@teurons/neptune-nodejs");
+
+const addUserToNeptune = async (uuid) => {
+	let neptuneData = {
+		user_id: uuid,
+	};
+	await neptune.createUser(neptuneData);
+};
+
+const prepareUserContactInfoPayload = async (payload) => {
+	let neptuneData = {};
+
+	switch (payload.type) {
+		case "email":
+			neptuneData = {
+				type: "email",
+				value: payload.value,
+				filterable: "regular",
```

Comment:
Do we need this "regular" here? I think we don't need this.

Reply:
i followed Neptune's documentation there.

Reply:
@rjvim made changes please review now.